src/components/notification/NotificationsWrapper.js

```diff
@@ -1,12 +1,17 @@
 // @flow
 import * as React from 'react';
 
+import FocusTrap from '../focus-trap';
 import Portal from '../portal';
 
 type Props = {
     children?: React.Node,
 };
 
-const NotificationsWrapper = ({ children }: Props) => <Portal className="notifications-wrapper">{children}</Portal>;
+const NotificationsWrapper = ({ children }: Props) => (
+    <Portal className="notifications-wrapper" aria-live="polite">
```

Comment:
It seems from the docs for `aria-live` that the content has to always be there in the DOM for it to read it out. Since this appearing in a transient way, does this work with screen readers?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

"Always make sure that the live region is present in the document first, and only then dynamically add/change any content."

Reply:
<NotificationsWrapper> is always present and aria attributes are applied to it so this works fine. 
I did experience weird behavior when I applied live region to individual notification. Which makes sense now given the docs you shared.